tests/dashboards/security/security_dataset_tests.py

```diff
@@ -208,20 +208,6 @@ def test_get_dashboards__users_can_view_permitted_dashboard(self):
         finally:
             self.revoke_public_access_to_table(accessed_table)
 
-    def test_get_dashboard_api_no_data_access(self):
-        """
-        Dashboard API: Test get dashboard without data access
-        """
-        admin = self.get_user("admin")
-        dashboard = create_dashboard_to_db(
-            random_title(), random_slug(), owners=[admin]
-        )
-
-        self.login(username="gamma")
-        uri = DASHBOARD_API_URL_FORMAT.format(dashboard.id)
-        rv = self.client.get(uri)
-        self.assert404(rv)
-
```

Comment:
This is a duplicate test to https://github.com/apache/superset/blob/7a9036e35d557fccd0ccaeb7277c10b9e1039a70/tests/dashboards/api_tests.py#L221